views/bootlint.pug

```diff
@@ -14,7 +14,7 @@ block content
             .input-group.input-group-lg
                 input.form-control(id=formId, type='text', value=file)
                 span.input-group-btn
-                    button.btn.btn-secondary(type='button', data-toggle='collapse', data-target='#' + name, aria-expanded='false', aria-controls=name)
+                    button.btn.btn-dark(type='button', data-toggle='collapse', data-target='#' + name, aria-expanded='false', aria-controls=name)
```

Comment:
@XhmikosR what is the diff? I don't see anything visually.

Reply:
Well, I shouldn't have used `secondary` in the first place, since we wanted dark. It just happened to look the same for the theme we were using.